Firebase/Core/FIRApp.m

```diff
@@ -430,6 +451,18 @@ + (NSError *)errorForInvalidAppID {
                          userInfo:errorDict];
 }
 
++ (void)registerAsConfigurable:(Class<FIRCoreConfigurable>)klass {
+  // This is called at +load time, keep the work to a minimum.
+  static dispatch_once_t onceToken;
+  dispatch_once(&onceToken, ^{
+    gRegisteredAsConfigurable = [[NSMutableArray alloc] initWithCapacity:1];
+  });
+
+  NSAssert([(Class)klass conformsToProtocol:@protocol(FIRCoreConfigurable)],
```

Comment:
Is this check redundant? Won't it be checked at compile time in the method signature?

Reply:
This was not getting caught for me when integrating Auth. I'm not sure exactly why, but I got no warnings at compile time.